### DISCUSSION TITLE
Cannot define a GATv2Conv layer for a Bipartite graph

### DISCUSSION THREAD

**Discussion by michalisfrangos** [pyg-team/pytorch_geometric#3838] [General]
Hello,

From the description of GATConv and GATv2Conv they both support bipartite graphs and the layers can be instantiated in the same way. I cannot define a layer for a Bipartite graph with GATv2Conv. I was wondering if this is a bug in GATv2Conv? 
Please see additional info below. 

Thanks
M

Example with GATConv (all good):
```
GATConv(in_channels = (10,10),
          out_channels = 32,
          heads = 1, concat=True)
```
gives a layer` GATConv((10, 10), 32, heads=1)`

Example with GATv2Conv (error):
```
GATv2Conv(in_channels = (10,10),
          out_channels = 32,
          heads = 1, concat=True)
```
gives an error:
`---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
~\AppData\Local\Temp/ipykernel_12336/3079787839.py in <module>
----> 1 GATv2Conv(in_channels = (10,10),
      2           out_channels = 32,
      3           heads = 1, concat=True)

~\miniconda3\envs\env_gnn\lib\site-packages\torch_geometric\nn\conv\gatv2_conv.py in __init__(self, in_channels, out_channels, heads, concat, negative_slope, dropout, add_self_loops, bias, share_weights, **kwargs)
     80         self.share_weights = share_weights
     81 
---> 82         self.lin_l = Linear(in_channels, heads * out_channels, bias=bias,
     83                             weight_initializer='glorot')
     84         if share_weights:

~\miniconda3\envs\env_gnn\lib\site-packages\torch_geometric\nn\dense\linear.py in __init__(self, in_channels, out_channels, bias, weight_initializer, bias_initializer)
     47         self.bias_initializer = bias_initializer
     48 
---> 49         if in_channels > 0:
     50             self.weight = Parameter(torch.Tensor(out_channels, in_channels))
     51         else:

TypeError: '>' not supported between instances of 'tuple' and 'int'`

**Comment by rusty1s** [comment]
Which version of PyG are you using (`torch_geometric.__version__`)? This should be already fixed in version `2.0.3`.

**Reply by michalisfrangos** [reply]
got it! I was using 2.0.1....   all good in version 2.0.3.
